superset-frontend/src/components/ListView/ListView.test.jsx

```diff
@@ -462,7 +470,7 @@ describe('ListView', () => {
     });
 
     await act(async () => {
-      wrapper2.find('[data-test="card-sort-select"]').first().props().onChange({
+      wrapper2.find('[aria-label="Sort"]').first().props().onChange({
```

Comment:
nit: should this be lower case?

Reply:
We are now referring to the `aria-label` property, which doesn't have to be lowercase.